test/js/bun/glob/stress.test.ts

```diff
@@ -1,26 +1,26 @@
 import { expect, test, describe, beforeAll } from "bun:test";
 import { Glob } from "bun";
 import { tempFixturesDir } from "./util";
-
+import path from "path";
 const paths = [
-  "test/js/bun/glob/fixtures/file.md",
-  "test/js/bun/glob/fixtures/second/file.md",
-  "test/js/bun/glob/fixtures/second/nested/file.md",
-  "test/js/bun/glob/fixtures/second/nested/directory/file.md",
-  "test/js/bun/glob/fixtures/third/library/b/book.md",
-  "test/js/bun/glob/fixtures/third/library/a/book.md",
-  "test/js/bun/glob/fixtures/first/file.md",
-  "test/js/bun/glob/fixtures/first/nested/file.md",
-  "test/js/bun/glob/fixtures/first/nested/directory/file.md",
-  "test/js/bun/glob/fixtures/first/nested/directory/file.json",
+  path.join(import.meta.dir, "fixtures/file.md"),
+  path.join(import.meta.dir, "fixtures/second/file.md"),
+  path.join(import.meta.dir, "fixtures/second/nested/file.md"),
+  path.join(import.meta.dir, "fixtures/second/nested/directory/file.md"),
+  path.join(import.meta.dir, "fixtures/third/library/b/book.md"),
+  path.join(import.meta.dir, "fixtures/third/library/a/book.md"),
+  path.join(import.meta.dir, "fixtures/first/file.md"),
+  path.join(import.meta.dir, "fixtures/first/nested/file.md"),
+  path.join(import.meta.dir, "fixtures/first/nested/directory/file.md"),
+  path.join(import.meta.dir, "fixtures/first/nested/directory/file.json"),
 ];
 
 beforeAll(() => {
   tempFixturesDir();
 });
 
 test("Glob.scan stress test", async () => {
-  const cwd = "test/js/bun/glob";
+  const cwd = import.meta.dir;
```

Comment:
@zackradisic the cwd is the `test` folder in the process, not the `bun` directory. Did you manually edit bunfig.toml to make it not the `test` directory?